pkg/kapp/clusterapply/delete_change.go

```diff
@@ -122,3 +139,12 @@ func descMessage(res ctlres.Resource) []string {
 	}
 	return []string{}
 }
+
+func isInoperableResource(res ctlres.Resource) bool {
```

Comment:
We should scope this function to DeleteChange: `func (c DeleteChange) isInoperableResource(...`

Reply:
updated